src/routes/public.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Home from '../screens/Home';
+import Login from '../screens/Login';
+
+import { loginScreen } from '../constants/navigation/constants';
+import { homeScreen } from '../constants/navigation/constants';
+
+type StackParamList = {
+  Home: undefined;
+  Login: undefined;
+};
+
+const Stack = createNativeStackNavigator<StackParamList>();
+
+const Routes = () => {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator
+        initialRouteName={homeScreen}
+        screenOptions={{
+          headerStyle: {
+            backgroundColor: '#222222',
+          },
+          headerTintColor: '#FFF',
+        }}>
+        <Stack.Screen name={homeScreen} component={Home} />
+        <Stack.Screen name={loginScreen} component={Login} />
+      </Stack.Navigator>
+    </NavigationContainer>
```

Comment:
...e aqui mudaria para
```suggestion
    <NavigationContainer>
      <Stack.Navigator
        initialRouteName={routes.HOME}
        screenOptions={{
          headerStyle: {
            backgroundColor: '#222222',
          },
          headerTintColor: '#FFF',
        }}>
        <Stack.Screen name={routes.HOME} component={HomeScreen} />
        <Stack.Screen name={routes.LOGIN} component={LoginScreen} />
      </Stack.Navigator>
    </NavigationContainer>
```